src/services/event-service.ts

```diff
@@ -110,6 +121,26 @@ export function eventService(
       (method) => method.priorityIdentifier === "BACKUP"
     );
 
+    const defaultPhoneNumberCountryCodeObject =
+      defaultMethod?.method.mfaMethodType === mfaMethodTypes.sms
+        ? {
+            phone_number_country_code: parsePhoneNumber(
+              defaultMethod.method.phoneNumber,
+              "GB"
```

Comment:
Hi @saralk - does phone_number_country_code always get returned properly for international phone numbers as well considering we are hard coding `GB`?

Reply:
@TaherKapasi the "GB" parameter is the default country code, if there is no international dialing code (e.g. +33), we assume GB.

https://www.npmjs.com/package/libphonenumber-js#api